docs/libs/rlottie.rst

```diff
@@ -163,13 +167,14 @@ wonderful utility to LVGL on embedded LCDs and can look really good when
 done properly.
 
 When picking/designing a lottie animation consider the following
-limitations: - Build the lottie animation to be sized for the intended
-size - it can scale/resize, but performance will be best when the base
-lottie size is as intended - Limit total number of frames, the longer
-the lottie animation is, the more memory it will consume for rendering
-(rlottie consumes IRAM for rendering) - Build the lottie animation for
-the intended frame rate - default lottie is 60fps, embedded LCDs likely
-won't go above 30fps
+limitations:
+
+- Build the lottie animation to be sized for the intended size - it can scale/resize, 
```

Comment:
This happened to me a lot when i was going over the files to correct things. 

it should be 
```
- Build the lottie animation to be sized for the intended size 
- it can scale/resize,
```

Reply:
fixed